book/src/07_rendering.md

```diff
@@ -0,0 +1,54 @@
+# Rendering
+
+Rendering is often a little bit tricky when you're dealing with a multi-threaded ECS.
+That's why we got something called "thread-local systems".
+
+There are two things special about thread-local systems:
+
+1) They're always executed at the end
+2) They cannot have dependencies, you just add them in whatever order you want
```

Comment:
Is "whatever order you want" also the order in which these are run? In that case I think this sentence could be made more clear. Maybe "you just add them in the order you want them to run".